app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1659,6 +1688,23 @@ BrowserWorld::CreateSkyBox(const std::string& aBasePath, const std::string& aExt
   }
 }
 
+void BrowserWorld::Create6DOFEnvironment() {
+  ASSERT_ON_RENDER_THREAD();
+  m.rootEnvironment = Transform::Create(m.create);
+  m.rootEnvironment->AddLight(Light::Create(m.create));
+
+  vrb::TransformPtr model = Transform::Create(m.create);
+  m.loader->LoadModel("FirefoxPlatform2_low.obj", model);
+  m.rootEnvironment->AddNode(model);
+  vrb::Matrix transform = vrb::Matrix::Identity();
+  model->SetTransform(transform);
+
+  m.layerEnvironment = m.device->CreateLayerProjection(VRLayerSurface::SurfaceType::FBO);
+  if (m.layerEnvironment) {
+    m.rootEnvironment->AddNode(VRLayerNode::Create(m.create, m.layerEnvironment));
+  }
+}
+
```

Comment:
I'm not sure 6DOF is a good description. I assume we will enable this on devices that are not 6DoF. Since we already call the current envs skyboxes, I would be okay just calling it the environment.

Reply:
makes sense, I changed the name